README.md

```diff
@@ -312,6 +312,38 @@ The following sample plugins exist for NRI:
 Please see the documentation of these plugins for further details
 about what and how each of these plugins can be used for.
 
+## Protecting Access to NRI
```

Comment:
I'd suggest having more general title, because this discussion has a broader view than just accessing NRI.

Maybe just `## Security` would be good? Later on, some other security considerations will fit under this topic, too.

Reply:
Updated to 'Security Considerations'.